src/lib/structures/Josh.ts

```diff
@@ -700,40 +716,31 @@ export class Josh<StoredValue = unknown> {
 
     payload.trigger = Trigger.PostProvider;
 
+    /* istanbul ignore if */
     if (payload.error) throw payload.error;
 
     for (const middleware of this.middlewares.array()) await middleware.run(payload);
     for (const middleware of this.middlewares.getPostMiddlewares(Method.Find)) payload = await middleware[Method.Find](payload);
 
     if (isPayloadWithData<Record<string, StoredValue>>(payload)) return payload.data;
 
+    /* istanbul ignore next: covered in provider tests */
     throw this.providerFailedError;
   }
 
   /**
-   * Get a value using a key.
+   * Get a value using a key and/or path.
    * @since 2.0.0
    * @param key A key at which a value is.
+   * @param path A path to the value.
    * @returns The value gotten or null.
    *
    * @example
    * ```javascript
    * await josh.set('key', 'value');
    *
    * await josh.get('key'); // 'value'
-   * ```
-   */
-  public async get(key: string): Promise<StoredValue | null>;
```

Comment:
why is this overload removed?

Reply:
It isn't required afaik and it caused errors with more than 1 parameter

Reply:
It is because this forces the return value to `StoredValue`

Reply:
Well when path is provided, the overload makes ts error that it only expected 1 argument, so is that something you can look at? I don't think I understand TS overloads enough in this area

Reply:
this should be fixed now